digdag-core/src/main/java/io/digdag/core/agent/InProcessTaskServerApi.java

```diff
@@ -7,11 +7,28 @@
 import io.digdag.spi.TaskQueueClient;
 import io.digdag.core.queue.TaskQueueServerManager;
 import io.digdag.core.workflow.WorkflowExecutor;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Comparator;
 import java.util.List;
+import java.util.function.ToLongFunction;
+import java.util.stream.Collectors;
 
 public class InProcessTaskServerApi
     implements TaskServerApi
 {
+    private static final Logger logger = LoggerFactory.getLogger(WorkflowExecutor.class);
+    private final ToLongFunction<TaskQueueLock> CONV_FUNC_FROM_TASK_QUEUE_LOCK_TO_INT = taskQueueLock -> {
```

Comment:
I am curious the naming rule of the function.
Is there any reference to use capital letter with _ for the var name of function ?

Reply:
Oops. I should have made this `static` since this variable is a constant... I'll fix it

Reply:
fyi: `Constants` on https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html just in case